chihacknight/decarbonize-my-state
# Show Public Transit + Micromobility Transit Suggestions & Optimize Images

## Overview

Adds the new streetcar and e-bike assets from our artist, Josephine, and moves the state details transport section to have two electric cars and then the street car and e-bike. This is related to my feelings that we were still focusing on electric cars too much, and public transit and ebikes overall still have much _much_ less emissions per mile, so encouraging folks towards those is also very beneficial.

| Before | After |
| --- | --- |
| ![Screenshot from 2023-02-20 23-05-07](https://user-images.githubusercontent.com/3187531/220255024-4a9a70fd-7ad4-49ae-ab9b-64e2c17317da.png) | ![Screenshot from 2023-02-20 23-05-13](https://user-images.githubusercontent.com/3187531/220255020-1c63dc09-0fa3-4861-beb1-cf4e5d811ec5.png) |

Closes _none_ (no related issue)

### Demo

See description for visual diff.

**Performance Comparison:**

| Before | After | 
| --- | --- |
| ![Screenshot from 2023-02-20 23-23-09](https://user-images.githubusercontent.com/3187531/220254969-fca0bdb6-17a3-413b-91bf-a798002b53ac.png) | ![Screenshot from 2023-02-20 23-23-18](https://user-images.githubusercontent.com/3187531/220254965-c53d476a-b960-455e-bba7-7012bdcd6d52.png) |

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

- Confirm state details page transport section looks good on desktop and mobile
- Confirm other pages with vehicles look good (since the whole sheet was changed)
- Confirm that images are indeed smaller but still look high quality
